app/gui2/src/components/visualizations/SQLVisualization.vue

```diff
@@ -4,7 +4,7 @@ export const inputType = 'Standard.Database.Data.Table.Table | Standard.Database
 export const defaultPreprocessor = [
   'Standard.Visualization.SQL.Visualization',
   'prepare_visualization',
-]
+] as const
```

Comment:
are these `as const`s really needed?

Reply:
It makes the type true to what's specified in `VisualizationModule`. I attempted to remove the `as any` from `builtinVisualizationImports` list, and I sort of succeeded with it, at least in IDE. Unfortunately it still breaks during external typecheck, so I've kept the any casts for now. But at least I've made some progress towards that.